src/js/components/Data/stories/Inline.js

```diff
@@ -33,6 +34,7 @@ const Filters = ({ search, ...rest }) => (
   <DataFilters {...rest}>
     {search && <DataSearch property="sub.name" />}
     <DataFilter property="size" />
+    <DataTableGroupBy options={['size']} />
```

Comment:
In the case where there is only 1 item in each group it feels weird to show the expand/collapse control

https://user-images.githubusercontent.com/54560994/232119991-8f48b121-1438-4791-8d2e-2b111f279a09.mp4

Reply:
I've added a table header and more rows.